src/Integration.Vsix/BoundSolutionAnalyzer.cs

```diff
@@ -91,13 +91,12 @@ private void OnSolutionBuilding()
 
         private static string[] GetFiles(string rootDirectory, string folder, string pattern)
         {
-            string[] matches = null;
             string searchDir = Path.Combine(rootDirectory, folder);
-            if (Directory.Exists(searchDir))
+            if (!Directory.Exists(searchDir))
```

Comment:
My teacher in school would kill me because they say we should have only 1 IN and 1 OUT but I think this is so much more readable like this. I know that we exit there instead of having to remember state later on the method.